filer/models/imagemodels.py

```diff
@@ -44,14 +44,14 @@ class Image(File):
     date_taken = models.DateTimeField(_('date taken'), null=True, blank=True,
                                       editable=False)
 
-    default_alt_text = models.CharField(
-        _('default alt text'), max_length=255, blank=True, null=True,
+    default_alt_text = models.TextField(
+        _('default alt text'), blank=True, null=True,
         help_text=_('Describes the essence of the image for users who have '
                     'images turned off in their browser, or are visually '
                     'impaired and using a screen reader; and it is used to '
                     'identify images to search engines.'))
-    default_caption = models.CharField(
-        _('default caption'), max_length=255, blank=True, null=True,
+    default_caption = models.TextField(
+        _('default caption'), blank=True, null=True,
         help_text=_('Caption text is displayed directly below an image '
                     'plugin to add context; there is a 140-character limit,'
```

Comment:
We should also change the help text, as the 140-character limit is no longer valid. We don't have that limitation anymore on the UI.

Reply:
done